src/Microsoft.DotNet.Cli.Utils/CommandResolution/ProjectToolsCommandResolver.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.DotNet.Cli.Utils
 {
     public class ProjectToolsCommandResolver : ICommandResolver
     {
-        private static readonly NuGetFramework s_toolPackageFramework = FrameworkConstants.CommonFrameworks.NetStandardApp15;
+        private static readonly NuGetFramework s_toolPackageFramework = FrameworkConstants.CommonFrameworks.NetCoreApp10;
```

Comment:
A tool targeting `netstandardapp` will **already** be broken because of the upcoming NuGet change @pakrym referenced (https://github.com/NuGet/NuGet.Client/commit/9573f19ce94c84078e0d0e4f128bb24de7bfe140). NuGet only restores the tool for `netcoreapp1.0`. There's no point in having a preference order because NuGet will **never** put a `netstandardapp` tool, it will just fail with a compatibility error.